signalfx-smart-gateway/README.md

```diff
@@ -43,7 +43,16 @@ The downloaded checksum assumes that the Smart Gateway binary is in the current
 
 #### Changelog
 
-##### Latest Version: v1.1.1 (June 4, 2019)
+
+##### Latest Version: v1.1.2 (June 18, 2019)
+
+[Download Smart Gateway v1.1.2](https://app.YOUR_SIGNALFX_REALM.signalfx.com/v2/smart-gateway/download/v1.1.2)  
+_SHA256: `5842d98fc408c26ff8579e429f6f3cffa67cceca0305927dd18e578cb840cfad`_
```

Comment:
This sha256 checksum also needs to be updated:
9e078a1505170c035ee7d452096918d90b0449bb2f85a5b6f55c0999204a0926